modules/hub/src/vendor/connext/hasPendingOps.ts

```diff
@@ -0,0 +1,12 @@
+import { ChannelState } from './types'
+
+export function hasPendingOps(state: ChannelState) {
+  for (let field in state) {
+    if (!field.startsWith('pending'))
+      continue
+    if ((state as any)[field] != '0')
+      return true
+  }
+  return false
+}
+
```

Comment:
Are changes in `modules/hub/src/vendor/connext` still necessary? If so, is the client even really extracted?

Looks like we're importing from vendor rather than node_modules, I'm curious about why this is.

Reply:
Yep this PR overwrote your changes. We need to make this change from the Spank side so that we can keep the code consistent from there. We have a PR in there which we will bring into here when it gets merged.